openupgrade_scripts/scripts/analytic/18.0.1.2/post-migration.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2025 Hunki Enterprises BV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from openupgradelib import openupgrade, openupgrade_180
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.load_data(env, "analytic", "18.0.1.2/noupdate_changes.xml")
```

Comment:
```suggestion
    openupgrade.load_data(env, "analytic", "18.0.1.2/noupdate_changes.xml", mode="init_no_create")
```
This is because now the record has `forcecreate="0"`

Reply:
but that's fine. we don't want to create it if it doesn't exist, just rename it if it does